internal/net/ssl/ssl.go

```diff
@@ -230,10 +227,10 @@ func ConfigureCRL(name string, crl []byte, sslCert *ingress.SSLCert) error {
 
 	_, err := x509.ParseRevocationList(pemCRLBlock.Bytes)
 	if err != nil {
-		return fmt.Errorf(err.Error())
+		return errors.New(err.Error())
```

Comment:
This is weird for me, why we just don't return the error?

Reply:
Updated